test/run-php-linter.sh

```diff
@@ -18,4 +18,4 @@ vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php htdocs || exit
 # Run PHPCS on all modules
 vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/ || exit $?;
 
-vendor/bin/phpmd php/libraries text docs/LorisPHPMD.xml || exit $?;
+vendor/bin/phpmd php/,modules/ text docs/LorisPHPMD.xml || exit $?;
```

Comment:
This change makes it so that all sub-folders of `php/` as well as the `modules/` folder will be checked against our PHP Mess Detector ruleset. Previously only `php/libraries` was checked.

Reply:
what about the `src` directory?

Reply:
@johnsaigle what about the `src` directory?

Reply:
Oops I missed this. I can add the `src` directory.